Metro-Records/scrapers-lametro
# Update legislative session automatically

## Overview

This branch does two things. It determines the current fiscal year programmatically as opposed to having it hardcoded, and it allows for the next fiscal year to be scraped during and after the last week of the current fiscal year.

This means that on June 10th, 2024 (before the last week), the latest fiscal year that would be scraped would end on June 30th, **2024**. However once June 23rd, 2024 hits (the beginning of the last week), the latest year scraped would end on June 30, **2025** and would remain that way up until June 23 of 2025, wherein it would scrape up to '26, etc.

- Closes #22 

## Testing Instructions
* Run a scrape
  * `docker-compose run --rm scrapers pupa update lametro bills window=1 --rpm=0`
* Confirm that the scrape completes as expected
* Confirm that automated tests pass and that they are appropriately testing behavior
